core/standard/annex_stac.adoc

```diff
@@ -0,0 +1,237 @@
+[appendix]
```

Comment:
Would it make more sense to just link to the original document? I don't really want to maintain this information in two places and I think this is more a living document which evolves over time while we further align and evolve (e.g. STAC 1.1). Having it as part of a published standard doesn't feel right.

Reply:
16-OCT-2023 SWG Meeting: In general the SWG believes that it would be better to link out to this information.  However, it was copied into the standard because we are not sure about the persistence of the source material.  @pvretano will ping @m-mohr about that and update the document accordingly. @kalxas mentioned that the source material is in the [STAC UTILS](https://github.com/stac-utils) repo and might be better if it was closer to the [STAC specification](https://github.com/radiantearth/stac-spec) repo.

Reply:
I have access to stac-spec, but we try to keep non-normative separate, we'll even try move the best practices out. So it's unlikely that the comparison document will move to stac-spec. Also, this was never reviewed by the STAC PSC, it was just me writing this as part of another project. What's the concern here? That the status is not stable (e.g. we may fix issues over time)? Or that it just goes offline at some point?